test/DynamoCoreWpfTests/SplashScreenTests.cs

```diff
@@ -1,13 +1,34 @@
 
 
+using DynamoCoreWpfTests.Utility;
 using DynamoUtilities;
 using NUnit.Framework;
+using System;
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+using System.Threading;
+using System.Threading.Tasks;
 
 namespace DynamoCoreWpfTests
 {
     [TestFixture]
     internal class SplashScreenTests
     {
+        public enum WindowsMessage
+        {
+            WM_CLOSE = 0x0010
+        }
+
+        [DllImport("user32.dll")]
+        public static extern IntPtr FindWindow(string lpClassName, String lpWindowName);
+
+        [DllImport("user32.dll")]
+        public static extern int SendMessage(IntPtr hWnd, int wMsg, IntPtr wParam, IntPtr lParam);
+
+        [DllImport("user32.dll", SetLastError = true, CharSet = CharSet.Auto)]
+        static extern uint RegisterWindowMessage(string lpString);
```

Comment:
is this used?

Reply:
No, it was during my first pass of writing the test.
I'll clean it up in master